game-core/src/main/java/games/strategy/triplea/ui/UserActionPanel.java

```diff
@@ -192,20 +191,6 @@ static boolean canPlayerAffordUserAction(final PlayerID player, final UserAction
     return userAction.getCostPu() <= player.getResources().getQuantity(Constants.PUS);
   }
 
-  private static Dimension getUserActionScrollPanePreferredSize(final JScrollPane scrollPane) {
```

Comment:
According to veqryn this code was there to account for any potential scrollbar that appears when the content is too big and causes another scroll bar to appear, so a second scrollbar doesn't appear just because the content no longer fits in the frame once the first scrollbar takes up space.
However this didn't seem to work so I dropped the code.